src/main/java/org/apache/commons/text/cases/KebabCase.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.commons.text.cases;
+
+/**
+ * Case implementation which parses and formats strings of the form 'my-kebab-string'
+ * <p>
+ * KebabCase is a delimited case where the delimiter is a hyphen character '-'.
+ * </p>
+ */
+public final class KebabCase extends CharacterDelimitedCase {
```

Comment:
I'm not opposed to that. Would it make sense to place them in a CasesUtils class along side something like a util convert method...

```
public static final KebabCase KEBAB = KebabCase.INSTANCE;
public static final SnakeCase SNAKE = SnakeCase.INSTANCE;
public static final CamelCase CAMEL = CamelCase.INSTANCE;
public static final PascalCase PASCAL = PascalCase.INSTANCE;

public static String convert(String string, Case from, Case to) {
        return to.format(from.parse(string));
}
```

Not sure if that has any benefit since it's just replacing one line of code, but might be useful in regards to method references in new Java version?